### PR TITLE
Rewrite with nbextension

### LINKED ISSUE #20: Use npm instead of submodules
Vega, vega-lite, polestar and voyager are on npm or bower. So we should use those.

### PR DESCRIPTION
**Pull request by domoritz**
This branch implements support for Vega (#18) and Vega-Lite (#10) but removes support for polestar and voyager for now. 

Fixes #20

### CODE REVIEW

**Comment by domoritz** [comment]
cc @ellisonbg

File: vega/utils.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import os.path
+import cgi
+import json
```
Comment (SamPenrose) [inline vega/utils.py]: Not used

File: vega/utils.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        else:
+            d[k] = u[k]
+    return d
+
+
+def scripts(paths):
+    """ Generate script tags for the given path """
+    out = []
+    for path in paths:
+        out.append(script(path))
```
Comment (SamPenrose) [inline vega/utils.py]: script is undefined

File: vega/vega.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+from IPython.display import display
+from ipywidgets import HTML
```
Comment (SamPenrose) [inline vega/vega.py]: Unused

File: vega/vegalite.py [hunk 4]
```diff
@@ -1,36 +1,48 @@
-from IPython import display
+from IPython.display import display
+from ipywidgets import HTML
```
Comment (SamPenrose) [inline vega/vegalite.py]: Unused

**Comment by domoritz** [comment]
@SamPenrose All cleaned up. Thanks for the comments.

**Comment by domoritz** [comment]
@jakevdp @ellisonbg I make this branch python 3 compatible and decided to merge it. Support for Vega-Lite and Vega seems more valuable at this point than hacky support for Voyager and Polestar.

**Pull request merged**